client_test.go

```diff
@@ -479,8 +477,12 @@ func TestGetFeature(t *testing.T) {
 	// Given
 	masterAPIKey := "master_api_key"
 	featureUUID := "10421b1f-5f29-4da9-abe2-30f88c07c9e8"
+	projectUUID := "10421b1f-5f29-4da9-abe2-30f88c07c9e8"
+
+	mux := http.NewServeMux()
+
+	mux.HandleFunc("/api/v1/features/get-by-uuid/10421b1f-5f29-4da9-abe2-30f88c07c9e8/", func(rw http.ResponseWriter, req *http.Request) {
```

Comment:
Is there a reason we're not using the `featureUUID` variable here and just hardcoding the UUID again?

Reply:
Yeah, I have moved them to constants